packages/main/src/MessageView.ts

```diff
@@ -0,0 +1,246 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import slot from "@ui5/webcomponents-base/dist/decorators/slot.js";
+import event from "@ui5/webcomponents-base/dist/decorators/event.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import SegmentedButton, { SegmentedButtonSelectionChangeEventDetail } from "./SegmentedButton.js";
+import MessageViewItem from "./MessageViewItem.js";
+import ListItemType from "./types/ListItemType.js";
+
+import MessageViewTemplate from "./generated/templates/MessageViewTemplate.lit.js";
+
+// Styles
+import MessageViewCss from "./generated/themes/MessageView.css.js";
+
+// Texts
+import {
+	MESSAGE_VIEW_MORE_INFORMATION,
+} from "./generated/i18n/i18n-defaults.js";
+import List from "./List.js";
+import Icon from "./Icon.js";
+
+enum MessageType {
+	Error = "Error",
+	Warning = "Warning",
+	Success = "Success",
+	Info = "Info"
+}
+
+enum MessageTypeIcon {
+	Info = "information",
+	Success = "status-positive",
+	Error = "error",
+	Warning = "status-critical"
+}
+
+enum DesignClassesMapping {
+	Info = "information",
+	Success = "success",
+	Error = "error",
+	Warning = "warning",
+}
+
+enum MessageViewMode {
+	// eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
why is this disabled?

Reply:
As discussed - renamed List enum type to Messages